packages/theme/src/theme/_guidelines.scss

```diff
@@ -87,8 +87,8 @@ $svg-sm-size: 1.2rem !default;
 
 /// Default button svg size
 /// @type Number (Unitless)
-$svg-md-size: 2rem !default;
+$svg-md-size: 1.6rem !default;
 
 /// Large button svg size
 /// @type Number (Unitless)
-$svg-lg-size: 2.5rem !default;
+$svg-lg-size: 2.4rem !default;
```

Comment:
Shouldn't you add the `$svg-xlg-size: 3.2 rem !default;` defined in the guideline too?